app/views/design-system/rsd-design-system/design-patterns/footer/index.html

```diff
@@ -0,0 +1,176 @@
+          &copy; Crown copyright
+        &lt;/a&gt;
+      &lt;/div&gt;
+    &lt;/div&gt;
+  &lt;/div&gt;
+&lt;/footer&gt;
+</code>' %}
+
+{% block beforeContent %}{% endblock %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    {{ sample.code('/design-system/rsd-design-system/design-patterns/footer/example.html', 'Footer', 'footer',
+    htmlCode) }}
+    <br>
+
+    <h2>When to use this component</h2>
+    <p>Use this pattern to create a product footer that is consistent with other RSD products.</p>
+    <figure class="govuk-!-margin-bottom-9" style="margin: 0;">
+      <img src="/assets/images/rsd-pattern-library-footer-screencapture-primary.png"
+        alt="A footer with Regional Services Division links." width="100%">
+      <figcaption>Screenshot of the RSD footer design pattern</figcaption>
+    </figure>
+
+    <h2>When not to use this component</h2>
+    <p>On</p>
+    <ul>
+      <li>external-facing sites, like <a href="https://apply-convert-academy.service.gov.uk/">Apply to become
+          an academy (A2B)</a> This site is for external users and the Microsoft links in this pattern
+        won't work for them</li>
+      <li>SharePoint guidance pages, like <a
+          href="https://educationgovuk.sharepoint.com/sites/lvewp00299/SitePages/Prepare-Conversions-and-Transfers.aspx">Prepare
+          conversions and transfers - SharePoint</a></li>
+    </ul>
+
+    <h2>Links</h2>
+    <p>The RSD footer includes several links:</p>
+    <h3>Email Service Support for help with using this system</h3>
+    <p>This is a mail-to link with the subject header:</p>
+    <code>[Product name]: support query</code>
+    <p>For example, Prepare conversions and transfers: support query</p>
+    <p> It has this text underneath the link: "We aim to respond within 5 working days, or one working day for
+      more urgent queries." </p>
+    <h3>How to use this system (opens in a new tab)</h3>
+    <p> This links to the section of a product's SharePoint page with guidance on how to use the product. For
+      example, <a
+        href="https://educationgovuk.sharepoint.com/sites/lvewp00299/SitePages/Prepare-Conversions-and-Transfers.aspx#guidance-for-using-prepare-conversions-and-transfers">Guidance
+        for using Prepare conversions and
+        transfers</a>. </p>
+    <p>If you don't have SharePoint guidance about how to use your product, you do not need to include this
+      hyperlink.</p>
+    <h3>Give feedback (opens in a new tab)</h3>
+    <p>This links to a Microsoft Form where users can give feedback about the product. The same Microsoft Form
+      is used across all RSD products: <a
+        href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-SZtRygfwTNOqcfRq-MXpv9UOTIyQlNYR0hJT1Q0TUFVSlJGVFhES01LVC4u">Give
```

Comment:
Consider opening in a new tab

Reply:
Thanks Andy. We had a discussion and since we decided not to open in a new tab for the "User needs database" (an internal Microsoft link), so for consistency, we also decided not to open this one in a new tab.